packages/metrics/src/calculateMetrics.ts

```diff
@@ -168,7 +170,7 @@ function countFileMetrics(fileResult: FileUnderTestModel[]): Metrics {
     totalValid,
     totalInvalid,
     mutationScore: totalValid > 0 ? (totalDetected / totalValid) * 100 : DEFAULT_SCORE,
-    totalMutants: totalValid + totalInvalid + ignored,
+    totalMutants: totalValid + totalInvalid + ignored + pending,
```

Comment:
pending should probably be added to totalValid instead of to totalMutants directly